wix-events-v2/wix-events-v2/guides/partial-updates.md

```diff
@@ -1,14 +1,16 @@
 # Partial Updates
-Most update endpoints support partial updates via a combination of a _partial entity body_ and a `fields` parameter.
+Most update functions support partial updates via a combination of a _partial entity body_ and a `fields` parameter.
```

Comment:
We generally don't italicize unless it's really necessary for emphasis. I think you can use standard formatting here.